src/preset.ts

```diff
@@ -1137,6 +1137,9 @@ async function tryApplyVsDevEnv(preset: ConfigurePreset) {
 
 async function expandConfigurePresetHelper(folder: string, preset: ConfigurePreset, workspaceFolder: string, sourceDir: string, allowUserPreset: boolean = false) {
     if (preset.__expanded) {
```

Comment:
A couple of questions,
1.  Why are we adding it in two locations that are basically the same? 
2. Is there not a place where we currently expand it relative to the binary directory as the bug says? If so, I would think that we need to fix that. 
3. Does the `util.resolvePath` handle if `installDir` is an absolute path? 

Instead of this, method, could we have the resolution in the overall `expandConfigurePreset` method?

Reply:
For 1) I could tell this was recursive and didn't know if they were both exit situation. I basically wanted to guarantee it's done after the presets have been completely expanded. 
For 2) This should handle that case too. I think that was just one case that happened, but the fix is to always set an absolute path like the CMake docs say
For 3) And yes, if installDir is an absolute path, it returns it how it was

Reply:
Your last point makes sense yeah. I'll fix the location

Reply:
updated @gcampbell-msft